### PR TITLE
Functional Tests : BO - Advanced Parameter - Authorization Server : Edit API Access (Enabled/Disabled)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : BO - Advanced Parameter - Authorization Server : Edit API Access (Enabled/Disabled)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/7273552596
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/pages/BO/advancedParameters/APIAccess/add.ts [hunk 1]
```diff
@@ -86,6 +86,12 @@ class AddNewAPIAccess extends BOBasePage {
   /*
   Methods
    */
+  async saveForm(page: Page): Promise<string> {
```
Comment (SD1982) [inline tests/UI/pages/BO/advancedParameters/APIAccess/add.ts:94]: Jsdocs is missing  here

File: tests/UI/pages/BO/advancedParameters/APIAccess/add.ts [hunk 2]
```diff
@@ -86,6 +86,12 @@ class AddNewAPIAccess extends BOBasePage {
...
+    // Save
+    await this.clickAndWaitForURL(page, this.saveButton);
+
+    return this.getAlertSuccessBlockParagraphContent(page);
+  }
 
   /**
    * Add API Access
```
Comment (SD1982) [inline tests/UI/pages/BO/advancedParameters/APIAccess/add.ts:102]: JsDocs is not complete here

**Review by SD1982 — DISMISSED** [review]

File: tests/UI/pages/BO/advancedParameters/APIAccess/add.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/pages/BO/advancedParameters/APIAccess/add.ts:94]: Fixed

File: tests/UI/pages/BO/advancedParameters/APIAccess/add.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/pages/BO/advancedParameters/APIAccess/add.ts:102]: Fixed

**Comment by Progi1984** [comment]
@SD1982 & @boubkerbribri Review fixed

**Review by boubkerbribri — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Pull request merged**